model/client.go

```diff
@@ -1483,6 +1483,16 @@ func (c *Client) GetPost(channelId string, postId string, etag string) (*Result,
 	}
 }
 
+func (c *Client) GetPostById(postId string, etag string) (*Result, *AppError) {
```

Comment:
Same as the other one, can we use `(*PostList, *ResponseMetadata)` as the return type and also add a comment explaining what this function does?